meson.build

```diff
@@ -103,6 +103,7 @@ if not get_option('source-only')
     sdl_options = ['default_library=static']
 
     # we explicitly need these
+    sdl_options += 'test=false'
```

Comment:
Should this be moved in the `# we don't need these` section?

Reply:
Done